dts/msm8226/msm8226-samsung-ms013g.dts

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id =  <0x9E00FF01 7 0x20000>,
```

Comment:
there should be only one that would be enough, like all other devices define it. Also the file name would follow the naming with that.

Reply:
i tried, but it rebooting in download mode with error